site/docs/v1/tech/apis/identity-providers/_oauth-idp-request-body.adoc

```diff
@@ -59,8 +85,49 @@ ifdef::idp_login_method[]
 include::_identity-provider-login-method-request-parameter.adoc[]
 endif::[]
 
+ifeval::["{idp_type}" == "Google"]
+[field]#identityProvider.properties# [type]#[Object]# [optional]#Optional# [since]#Available since 1.44.0#::
+An object to hold configuration parameters for the https://developers.google.com/identity/gsi/web/reference/html-reference[Google Identity Services HTML API].
+
+[field]#identityProvider.properties.api# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.44.0#::
+Google Identity Services HTML login API configuration in a properties file formatted String. Any attribute from https://developers.google.com/identity/gsi/web/reference/html-reference#element_with_id_g_id_onload[Google's documentation] can be added. Properties will be added to the HTML element with the `g_id_onload` element Id as data attributes. The properties specified in this field should not include the `data-` prefix on the property name.
```

Comment:
This isn't really limited to the HTML API - that is how we've shipped it though. But anyone could switch to the JavaScript API and use this same configuration. That was why I wanted to omit the `data-` and allow the user of the config to handle that part.

Reply:
So they would add a custom `<script>` tag to their template that iterates on these properties?